test/unit/view.child-views.spec.js

```diff
@@ -300,7 +300,7 @@ describe('layoutView', function() {
       var ChildView = Marionette.View.extend({
         template: false,
         onRender: function() {
-          this.triggerMethod('content:rendered');
```

Comment:
Not directly.  That'd be https://github.com/marionettejs/backbone.marionette/issues/2508

However if this were a childView of a `CollectionView` then `trigger` would work.  This PR is just about making sure that `trigger` works in both places.